packages/react-server-webpack-plugin/tsconfig.json

```diff
@@ -12,5 +12,5 @@
     "./src/**/*.tsx"
   ],
   "exclude": ["**/*.test.ts", "**/*.test.tsx"],
-  "references": [{"path": "../react-html"}, {"path": "../react-server"}]
+  "references": [{"path": "../react-server"}]
```

Comment:
Why is the `react-html` reference being removed? Did the script think it's not referenced?
It's definitely referenced in `packages/react-server-webpack-plugin/src/templates/client.ts`

Reply:
Because it's not part of the dependencies of the package. Probably a missing dependency?

Reply:
The `templates` folder is excluded from `eslint` so again this is the reason why the package is not mandatory in the `package.json`.

[Reference](https://github.com/Shopify/quilt/blob/master/.eslintignore#L6)

Reply:
I don't believe we use these templates anywhere anyways. This fine to leave as you have it now.